Examples/CrossBreeze_CrossTest_ExampleTests/App.config

```diff
@@ -106,8 +106,8 @@
 
 			<!-- The database servers which are used, the connectionStringName points to the name of the connectionStrings element defined in this app config. -->
 			<servers>
-				<server name="ExampleMsSqlServer" connectionName="ExampleMsSqlConnection" type="MsSql" commandTimeout="10" />
-				<server name="ExampleMsSsasServer" connectionName="ExampleMsSsasConnection" type="MsSsas" />
+				<server name="ExampleMsSqlServer" connectionName="ExampleMsSqlConnection" type="MsSql" commandTimeout="10" transactionIsolationLevel="Snapshot"/>

```

Comment:
I think isolationLevel would be enough right?

Reply:
I went for transactionIsolationLevel since it is only applied on the transactions (and not on all other SQL commands) but i changed it to isolationlevel since it is shorter.